tests/drivers/clock_control/stm32_clock_configuration/stm32_common_devices/boards/g0_i2c1_sysclk_lptim1_lsi.overlay

```diff
@@ -77,6 +77,6 @@
 };
 
 &adc1 {
-	/* Basic test only. Don't configure domain clock. */
+	/* Basic test only. ADC1 domain clock set by the board DTS */
```

Comment:
Let's be more explicit on the clock in use

Reply:
nucleo_g071rb DTS gives SYSCLK as clock source for the ADC1